modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -70,7 +70,7 @@
             <class name="org.wso2.identity.integration.test.entitlement.EntitlementRestServiceTestCase"/>
             <class name="org.wso2.identity.integration.test.requestPathAuthenticator.RequestPathBasicAuthenticationSSOTest" />
             <class name="org.wso2.identity.integration.test.idp.mgt.IdentityProviderMgtServiceTestCase" />
-            <class name="org.wso2.identity.integration.test.dashboard.IDENTITY5811TestCase" />
+            <!--<class name="org.wso2.identity.integration.test.dashboard.IDENTITY5811TestCase" />-->
```

Comment:
Have to fix test cases, rather commenting.

Reply:
These test cases are testing the dashboard...